content/security/for-admins/roles-and-permissions.md

```diff
@@ -95,3 +95,19 @@ The following outlines Docker Scout management permissions for member, editor, a
 | Activate and deactivate Docker Scout for a repository | ❌     | ✅     | ✅                 |
 | Create environments                                   | ❌     | ❌     | ✅                 |
 | Manage registry integrations                          | ❌     | ❌     | ✅                 |
+
+### Docker Build Cloud
+
+The following outlines Docker Build Cloud management permissions for member and organization owner roles.
+
+| Get Builder Account          | Member | Organization owner |
+| ---------------------------- | :----- | :----------------- |
+| Sign up for free plan        | ✅     | ✅                 |
+| View seat allocation         | ✅     | ✅                 |
```

Comment:
I'd just omit all `View $resource` entries when all roles even have write permissions on the same resource but that's just a suggestion